website/layouts/partials/versioning/banner.html

```diff
@@ -0,0 +1,9 @@
+{{- $currentVersion := index (split .Dir "/") 0 }}
+{{- $latestVersion := "tip" }}
+{{- $latestUrl := replace .URL $currentVersion $latestVersion }}
+
+{{- if ne $currentVersion $latestVersion }}
+<div class="alert alert-warning" role="alert">
+    <strong>This document is for an earlier version of Thanos. Click <a href="{{ $latestUrl }}">here</a> to see the latest version of this page.</strong>
```

Comment:
What happens if the current version does not have a document that an older version DOES have? The link will take you to a 404. Do we care about this case?

Reply:
I'm finding it hard to understand the context really. But I think it's okay if a current version does not have a document that an older version DOES have. I guess that's why we have versioning of the docs

Reply:
yes, this is an expected case, the question is HOW we respond to this case. In our example, we are essentially giving users bad advice. We are telling them: "hey, this is an old doc! take a look at the new version, which actually doesn't exist and will cause a 404." Maybe it would be better to be smart and check:
1. does the latest documentation have a newer version of this doc?
2. if yes, show the link
3. if not, tell the user that the doc is old but there is no new version

Does that make sense?

Reply:
Oh...I get your point now. Thanks for the clarifications 😍...would make some changes to this